Compiler/Program.cs

```diff
@@ -30,9 +30,17 @@ private static async Task WriteHelpText()
 
         private static async Task<int> Main()
         {
-            Log.Formatter = CommandLineFlags.FindLogFormatter(Environment.GetCommandLineArgs());
+            var args = new[]
+            {
+                //"--lang", "cs",
+                "--namespace", "Bebop.Codegen",
+                "--files", "C:/GitHub/ProgrammerAl/bebop/Laboratory/Schemas/Valid/jazz.bop",
+                "--cs", "C:/GitHub/ProgrammerAl/bebop/Laboratory/C#/GeneratedTestCode/Output.g.cs"
+            };
+
+            Log.Formatter = CommandLineFlags.FindLogFormatter(args);
 
-            if (!CommandLineFlags.TryParse(Environment.GetCommandLineArgs(), out _flags, out var message))
+            if (!CommandLineFlags.TryParse(args, out _flags, out var message))
```

Comment:
I don't think this was an intended change to commit.

Reply:
Woops, it was for local testing. I'll undo that change.